proto/properties.go

```diff
@@ -148,9 +148,9 @@ type Properties struct {
 	stype reflect.Type      // set for struct types only
 	sprop *StructProperties // set for struct types only
 
-	mtype    reflect.Type // set for map types only
-	mkeyprop *Properties  // set for map types only
-	mvalprop *Properties  // set for map types only
+	MapType  reflect.Type // set for map types only
+	MKeyProp *Properties  // set for map types only
+	MValProp *Properties  // set for map types only
```

Comment:
Call these `MapKeyProp` and `MapValueProp`?

Reply:
`MapKeyProp` & `MapValProp` seems a bit pretty